packages/autocertifier/protos/AutoCertifier.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+option optimize_for = CODE_SIZE;
+
+package autocertifier;
+
+message SessionIdRequest {
+    string sessionId = 1;
+}
+
+message SessionIdResponse {
+    optional string error = 1;
+    optional string sessionId = 2;
+}
+
+service AutoCertifierRpc {
+    rpc getSessionId (SessionIdRequest) returns (SessionIdResponse);
```

Comment:
We can only query existing sessions but not create new? Would it make sense to rename `SessionIdRequest` to `SessionRequest` (and response, and the  method)?

Reply:
Renamed to hasSession, HasSessionRequest, HasSessionResponse